airflow/providers/databricks/hooks/databricks.py

```diff
@@ -325,9 +331,9 @@ class _TokenAuth(AuthBase):
     magic function.
     """
 
-    def __init__(self, token):
+    def __init__(self, token: str) -> None:
         self.token = token
 
-    def __call__(self, r):
+    def __call__(self, r: PreparedRequest) -> PreparedRequest:
```

Comment:
May you provide a bit more info about this type? Say any link reference to requests doc relating to this? Thanks.

Reply:
you can find more info here: https://requests.readthedocs.io/en/master/api/#requests.PreparedRequest

If I am not wrong your request here it to provide a link in the comment (here) but not in the code!